.ci/gitlab/test.yml

```diff
@@ -29,6 +29,8 @@ stages:
 
 .test-common:
   extends: .test-common-local
+  # Run on shared runners
+  tags:
```

Comment:
I've probably written something like this myself before, but I had to double take on this before getting how this worked. Could you add to the comment that this overrides / empties the tags set in `.test-common-local`, hence the local runners not picking it up anymore (because they need `local`)?

Reply:
It is literally what was there before we moved off shared runners, and the most confusing thing to me is:
- We define `.test-common-local` which adds the tag `local`
- We derive `.test-common` from `.test-common-local` removing the tag

It makes much more sense to me to do the reverse, but I was minimizing the diff :-)

Reply:
yeah let's go with code clarity over minimal diffs :-)